gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sun Sep 27 10:59:54 BST 2015
+#Mon Nov 23 08:07:56 CST 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-all.zip
```

Comment:
No, my IDE actually did that. Please feel free to override that.

On Mon, Nov 30, 2015 at 6:13 AM, David Ferreira Pinto <
notifications@github.com> wrote:

> In gradle/wrapper/gradle-wrapper.properties
> <https://github.com/dpinto-pivotal/cf-SpringBootTrader/pull/43#discussion_r46136067>
> :
>
> >  distributionBase=GRADLE_USER_HOME
> >  distributionPath=wrapper/dists
> >  zipStoreBase=GRADLE_USER_HOME
> >  zipStorePath=wrapper/dists
> > -distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-bin.zip
> > +distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-all.zip
>
> Do we really need "all" rather than "bin"? it goes from 42MB to 62MB
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dpinto-pivotal/cf-SpringBootTrader/pull/43/files#r46136067>
> .
>



-- 
Corby Page
Senior Field Engineer, Pivotal
713.584.3168